### ISSUE TITLE
Accessibility (WCAG level AA) The API key privacy message doesn't have sufficient contrast against its background

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1171]
1.  Download Paciello Group's Contrast Analyser from http://www.paciellogroup.com/resources/contrastAnalyser.  This tool measures foreground-to-background contrast ratios using the same algorithm defined by the W3C's Web Content Accessibility Guidelines.
2.  Log in and navigate to https://nuget.org/account.  
3.  Compare the contrast ratio of the privacy message (Your key is hidden for privacy, click to show) on the page against its background.

Result:  It's 2.57:1.  
Expected:  This is below the threshold of 4.5:1 given in WCAG.  We should be able to fix this easily by slightly adjusting our color palette.

**Comment by johnataylor** [comment]
We have a new gallery design.
